PDFParse/pdfparse.ts

```diff
@@ -0,0 +1,39 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as pdf from 'pdf-parse';
+import * as readlineSync from 'readline-sync';
+
+async function convertPdfToTxt(): Promise<void> {
+  try {
+    // Get PDF file path from user input
+    const pdfPath = readlineSync.question('Enter the path to the PDF file: ');
+
+    // Check if the file exists
+    if (!fs.existsSync(pdfPath)) {
+      console.error('Error: The specified file does not exist.');
+      return;
+    }
```

Comment:
Getting files is gonna work a little differently that this but for testing this is file I guess. Rather then having the user enter the path we should just have a set path like /PDFParse/parse.pdf would be the default name.